src/store/reducer.js

```diff
@@ -0,0 +1,118 @@
+import {
+  SEARCH,
+  FILTER_MOVIE,
+  VISIBLE,
+  LOAD_MOVIES,
+  FILTER_ID,
+  REMOVE_MOVIE,
+  NEW_MOVIE,
+  ADD_MOVIE,
+  RESET,
+  LOAD_MOVIES_BY_TITLE,
+  LOAD_MOVIES_BY_DATE,
+  LOAD_MOVIES_BY_RATING,
+} from "./actionTypes";
+
+const initialState = {
+  movies: [],
+  searchFilter: "",
+  filterMovie: "",
+  movieArray: [],
+  visibility: true,
```

Comment:
we don't usually store data related to the UI on redux

Reply:
resolved